### DISCUSSION TITLE
[#995484] Arrows can be used to ignore Selection and Fill Limits

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#24] [Bugs]
if you use the arrows and drag the selection box manually you can make the selection box go thousands or more blocks then intended.

Video:
https://twitter.com/ihategravel2/status/1633560780531441667

**Comment by Landy005** [comment]
Although this is a bug, I, as creator, see use in this for me to load bedrock maps to export to java via chunker. Before we needed to place a block in every chunk to load an a square we could convert to java. Alot of people made systems that would generate platforms and then delete those just to register those chunks. But this seems like a good way for me to load a huge portion of the map where it is saved and ready for export!

In game: 
![Big_Selection_Stone](https://user-images.githubusercontent.com/16018643/223952636-b6bc1eda-6b64-4aa1-884d-d5a219b28fbb.png)

In chunker:
![Big_Selection_Stone_Chunker](https://user-images.githubusercontent.com/16018643/223952656-d353ae5e-d611-42a6-8680-568cf36c8d4b.png)

The only problem is that it doesnt fill the whole selection (probably because of limitations or it fills only what's rendered?). 
Perhaps a sliding bar for us to decide our own limits of selection and actions/operations would be better?

**Reply by ihategravel2** [reply]
Pretty sure it's not filling the entire selection because simulation distance.

**Reply by Landy005** [reply]
You're right, i noticed after aswell, though when we move ourselfes and redo the operation it does again fill the new simulated distance!
